docs/index.html

```diff
@@ -100,40 +104,107 @@ <h4>ヘッダー、フッターのメニュー</h4>
         <pre><code>
 <header>
   <h1><img src="..." alt=""></h1>
-  <nav class="p-drawer" aria-label="グローバルナビ">
-    <ul class="p-nav">
-      <li class="p-nav__item"><a href="..." class="p-nav__link">トップ</a></li>
-      <li class="p-nav__item"><a href="..." class="p-nav__link">サービス紹介</a></li>
-      <li class="p-nav__item"><a href="..." class="p-nav__link">企業情報</a></li>
-      <li class="p-nav__item"><a href="..." class="p-nav__link">採用情報</a></li>
+  <nav aria-label="グローバルナビ">
+    <ul>
+      <li><a href="...">トップ</a></li>
+      <li><a href="...">メンバー紹介</a></li>
+      <li><a href="...">配信アーカイブ</a></li>
+      <li><a href="...">泥酔情報</a></li>
     </ul>
   </nav>
 </header>
         </code></pre>
       </section>
+      <section>
+        <h4>同じ文言のリンク</h4>
+        <pre><code>
+<a href="..." aria-label="泥酔事例について詳しく見る">more...</a>
+<a href="..." aria-label="配信機材について詳しく見る">more...</a>
+<a href="..." aria-label="放送予定について詳しく見る">more...</a>
+        </code></pre>
+      </section>
       <section>
         <h3>aria-labelledby</h3>
-        <p>見出し要素を置けないようなコンテンツで、<br>他にラベルたるテキストがある場合、<br>そのid属性と<code>aria-labelledby</code>と紐付けられる</p>
+        <p>見出し要素を置けないようなコンテンツで、<br>他にラベルたるテキストがある場合、<br>そのid属性に<code>aria-labelledby</code>で紐付ける</p>
       </section>
       <section>
-        <h4>ラジオボタン</h4>
+        <h4>ラジオボタングループ</h4>
         <pre><code>
 <p id="optionLabel">泥酔するなら？</p>
 
-<div class="p-radioGroupe" role="radiogroup" aria-labelledby="optionLabel">
+<div role="radiogroup" aria-labelledby="optionLabel">
   <input type="radio" name="alcohol" value="...">ハイボール
   <input type="radio" name="alcohol" value="...">ストロングゼロ
   <input type="radio" name="alcohol" value="...">99.99
+  <input type="radio" name="alcohol" value="...">テキーラ
+</div>
+        </code></pre>
+      </section>
+      <section>
+        <h3>aria-controls</h3>
+        <p>自身が別要素の状態を制御していることを示す。</p>
+        <p>「状態」はJavaScriptで操作する。</p>
+        <p></p>
+      </section>
+      <section>
+        <h4>アコーディオン<br>（エキスパンダー／ディスクロージャ）</h4>
+        <pre><code>
+<h3>
+  <a
+    href="#disclosure1"
+    aria-label="企業情報のサブリンクを開く"
+    aria-controls="disclosure1"
+    aria-expanded="false"
+  >日本酒泥酔事件とは</a>
+</h3>
+<div id="disclosure1" aria-expanded="false">
```

Comment:
こっちに `aria-expanded` いらなさそう

```suggestion
<div id="disclosure1">
```

Reply:
えっそうだっけ

Reply:
トリガーの要素に `aria-expanded` と `aria-controls` があれば、展開される側にはいらないー

```html
<div aria-expanded="false">
  <button type="button">Open</button>
  <div aria-hidden="true">
    contents...
  </div>
</div>
```

or 

```html
<button type="button" aria-expanded="false" aria-controls="content">Open</button>
<div id="content" aria-hidden="true">
  contents...
</div>
```

という感じ。

https://www.w3.org/TR/wai-aria-practices-1.1/examples/disclosure/disclosure-img-long-description.html